ansible/roles/doorgeefluik/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: "Ensure doorgeefluik user exists"
+  user:
+    name: "doorgeefluik"
+    state: "present"
+    system: true
+    shell: "/bin/bash"
+  notify: "restart doorgeefluik"
+
+- name: "ensure clone location exists and is owned by doorgeefluik"
+  file:
+    path: "/var/www/doorgeefluik.{{ canonical_hostname }}"
+    state: "directory"
+    owner: "doorgeefluik"
+    group: "doorgeefluik"
+    recurse: true
+
+- name: "clone the repo"
```

Comment:
Preferably we run `nix-store --realize <derivation>` to fetch the build from cachix.
Because the current builds are not completely reproducible, we have to fetch the derivation path from GitHub to prevent rebuilds.
To fetch the builds from cachix, we have to run `cachix add svsticky-doorgeefluik` once

Reply:
Has been implemented with github releases